validator-network/src/network_impl.rs

```diff
@@ -90,14 +90,14 @@ where
     /// Look up the peer ID for a validator ID.
```

Comment:
Could you address the clippy warning above?

Reply:
Done.